src/averageAges.js

```diff
@@ -15,11 +20,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const targetSelector = century === undefined
+    ? person => (person.sex === 'm')
+    : person => (
+      person.sex === 'm'
+      && century === Math.ceil(person.died / 100)
+    );
```

Comment:
It seems that you have repeated code in both callbacks: `person.sex === 'm'`. Don't repeat yourself. You need only one filter function that checks gender && century (if it's needed) at once.